build.gradle

```diff
@@ -229,6 +232,8 @@ project(':ambry-admin') {
         testCompile project(':ambry-clustermap').sourceSets.test.output
         testCompile project(':ambry-rest').sourceSets.test.output
         testCompile project(':ambry-utils').sourceSets.test.output
+        testCompile project(':ambry-network').sourceSets.test.output
```

Comment:
might be able to get rid of this dependency too if we are able to move `SSLFactory` to ambry-commons